dataactcore/migrations/versions/87d17514eeed_job_status_constraints.py

```diff
@@ -0,0 +1,78 @@
+"""job_status_constraints
+
+Revision ID: 87d17514eeed
+Revises: 2701ef6ccb69
+Create Date: 2016-05-02 12:46:02.230844
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '87d17514eeed'
+down_revision = '2701ef6ccb69'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+
+
+def upgrade_error_data():
+    ### commands auto generated by Alembic - please adjust! ###
+    pass
+    ### end Alembic commands ###
+
+
+def downgrade_error_data():
+    ### commands auto generated by Alembic - please adjust! ###
+    pass
+    ### end Alembic commands ###
+
+
+def upgrade_job_tracker():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint(u'job_status_submission_id_fkey', 'job_status', type_='foreignkey')
+    op.create_foreign_key(None, 'job_status', 'submission', ['submission_id'], ['submission_id'], ondelete='CASCADE')
+    ### end Alembic commands ###
+
+
+def downgrade_job_tracker():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint(None, 'job_status', type_='foreignkey')
```

Comment:
I removed a similar line from an earlier migration because it causes the error "Can't emit DROP CONSTRAINT for ForeignKeyConstraint...None...it has no name".  I suspect that "None" is causing problems, the earlier version had the same thing.  Try upgrading to head and then downgrading past this version to see if you get the same error.  That said, dropping this particular constraint seems unnecessary as it's certainly not a bad thing for an earlier version to be cascading on delete, so we could just pass on the downgrade.